tests/index.js

```diff
@@ -0,0 +1,114 @@
+var spectron = require('spectron')
+var path = require('path')
+var tape = require('tape')
+
+// tape('init', function (t) {
+//   t.test('should be able to boot up the app', function (t) {
+//     var app = createApp()
+//     waitForLoad(app)
+//       .then(() => app.browserWindow.isVisible())
+//       .then((val) => t.equal(val, true))
+//       .then(() => app.client.getWindowCount())
+//       .then((val) => t.equal(val, 1, 'getWindowCount'))
+//       .then(() => app.browserWindow.isMinimized())
+//       .then((val) => t.equal(val, false, 'isMinimized'))
+//       .then(() => app.browserWindow.isDevToolsOpened())
+//       .then((val) => t.equal(val, false, 'isDevToolsOpened'))
+//       .then(() => app.browserWindow.isVisible())
+//       .then((val) => t.equal(val, true, 'isVisible'))
+//       .then(() => app.browserWindow.isFocused())
+//       .then((val) => t.equal(val, true, 'isFocused'))
+//       .then(() => app.browserWindow.getBounds())
+//       .then((val) => t.notEqual(val.width, 0, 'getBounds'))
+//       .then(() => app.browserWindow.getBounds())
+//       .then((val) => t.notEqual(val.height, 0, 'getBounds'))
+//       .then(() => endTest(app, t), (err) => endTest(app, t, err || 'error'))
+//   })
+// })
+
+tape('onboarding', function (t) {
+  // t.test('welcome screen should appear, and be dismissable', function (t) {
+  //   var app = createApp()
+  //   waitForLoad(app)
+  //     .then(() => app.browserWindow.isVisible())
+  //     .then((isVisible) => t.equal(isVisible, true))
+  //     .then(() => app.client.click('button'))
+  //     .then(() => app.browserWindow.getTitle())
+  //     .then((title) => t.equal(title, 'Dat Desktop'))
+  //     .then(() => endTest(app, t), (err) => endTest(app, t, err || 'error'))
+  // })
+
+  // t.test('welcome screen should show every time you open the app as long as you have no dats', function (t) {
+  //   var app = createApp()
+  //   waitForLoad(app)
+  //     .then(() => app.browserWindow.isVisible())
+  //     .then((isVisible) => t.equal(isVisible, true))
+  //     .then(() => app.client.click('button'))
+  //     .then(() => app.browserWindow.getTitle())
+  //     .then((title) => t.equal(title, 'Dat Desktop'))
+  //     .then(() => app.stop())
+  //     .then(() => Promise.resolve(app = createApp()))
+  //     .then(() => waitForLoad(app))
+  //     .then(() => app.browserWindow.isVisible())
+  //     .then((isVisible) => t.equal(isVisible, true))
+  //     .then(() => app.client.click('button'))
+  //     .then(() => app.browserWindow.getTitle())
+  //     .then((title) => t.equal(title, 'Dat Desktop'))
+  //     .then(() => endTest(app, t), (err) => endTest(app, t, err || 'error'))
+  // })
+
+  t.test('make sure you can minimize, full screen, resize, and move the window')
```

Comment:
probably don't need to test window resizing. more important to test key features that will prevent folks from using the app/ the app breaking entirely